src/app/modules/account/pages/registration/components/user-registration/user-registration.component.ts

```diff
@@ -28,9 +28,44 @@ import {Account} from "../../../../../../models/account.model";
   templateUrl: "./user-registration.component.html",
   styleUrls: ["./user-registration.component.scss"],
   standalone: true,
-  imports: [IonicModule, CommonModule, FormsModule],
+  imports: [IonicModule, CommonModule, FormsModule,ReactiveFormsModule],
 })
-export class UserRegistrationComponent {
+export class UserRegistrationComponent implements OnInit{
   @Input() account?: Partial<Account>;
-  constructor() {}
+  registrationForm: FormGroup;
+
+  constructor(private fb: FormBuilder) {
+    this.registrationForm = this.fb.group({
+      name: ['', Validators.required],
+      tagline:[''],
+      Description :[''],
+      phoneNumber:['', Validators.pattern(/^-?\d+$/)],
+      alternateEmail:['',[Validators.email]],
+      mailingAddress:[''],
+      emergencyContactNumber:['', Validators.pattern(/^-?\d+$/)],
+      preferredMethodOfContact :[''],
+      occupation:[''],
+      employerName:[''],
+      workExperience:[''],
+      skillsAndExpertise:[''],
+      currentJobTitle:[''],
+      linkedInProfile:[''],
+      educationalBackground:['']
+    });
```

Comment:
Can you have a look at how the `edit-user-profile` files are doing this.
Place any `user` specific details into the `userDetails` section, or if you're not sure you can leave it on the `account` object.  I'll see if @Nyero2023 or @quakermiss can have a go through these objects and figure out where these specific fields should go.
```
  editAccountForm = this.fb.group({
    name: ["", Validators.required],
    email: ["", [Validators.required, Validators.email]],
    description: [""],
    tagline: [""],
    language: [""],
    userDetails: this.fb.group({
      dateOfBirth: [new Date().toISOString(), [Validators.required]],
    }),
    address: this.fb.group({
      name: [""],
      street: ["", Validators.pattern("^[a-zA-Z0-9\\s,]*$")],
      city: ["", Validators.pattern("^[a-zA-Z\\s]*$")],
      state: ["", Validators.pattern("^[a-zA-Z\\s]*$")],
      zipcode: ["", Validators.pattern("^[0-9]*$")],
      country: ["", Validators.pattern("^[a-zA-Z\\s]*$")],
    }),
    phone: this.fb.group({
      countryCode: ["", [Validators.pattern("^[0-9]*$")]],
      number: ["", [Validators.pattern("^\\d{10}$")]],
      type: [""],
    }),
  });
  ```

Reply:
Let me know when you are ready for me or @Nyero2023 to review.  Thanks!

Reply:
I'm going to resolve this comment as I think we'll need to copy over the `group-registration` component for this component and base this component on that, as it's now fully built and has all the needed functionality for this component.